### PR TITLE
Package and ship locally

### PR DESCRIPTION
**Pull request by Riscky**
This removes the shipping step in CI.
Instead, the shipping script can now be run locally by anyone with access to the archive server.
This because we disabled public ssh access for this server.

### CODE REVIEW

File: package/build-and-ship.sh [hunk 1]
```diff
@@ -1,15 +1,10 @@
 #!/bin/bash
 
-# Entrypoint for building a package on Semaphore CI and shipping to our apt
-# file host. Only builds a package when the current build is a git tag.
+# Build Hoff and ship it to Freight. Only builds a package when the current
+# build is a git tag.
```
Comment (bertptrs) [inline package/build-and-ship.sh:4]: This statement appears not to be enforced anymore.

File: package/build-and-ship.sh [hunk 2]
```diff
@@ -21,13 +16,8 @@ export VERSION
...
-if [[ "$SEMAPHORE_GIT_REF_TYPE" != "tag" ]]; then
-    echo "Not on a tagged build. Skipping ship step."
-    exit 0
-fi
-
 PKGFILE="hoff_$VERSION-1.deb"
-FREIGHT_HOST="freight@archive-external.channable.com"
+FREIGHT_HOST="archive-petrol"
 
 scp "$PKGFILE" "$FREIGHT_HOST:/tmp/$PKGFILE"
```
Comment (bertptrs) [inline package/build-and-ship.sh:22]: This does not work as-written, I think. `gcloud compute scp --tunnel-through-iap` would be required. Similarly for the SSH command below.

**Review by bertptrs — CHANGES_REQUESTED** [review]
Left two comments regarding the new ship file. There are also some other package files; can we reduce the noise in the `package` dir?

File: package/build-and-ship.sh (hunk 2 above)
Comment (Riscky) [reply package/build-and-ship.sh:22]: It does work for people who've provisioned `gcloud_ssh_config` on their machine :p
I'll change it though (and also the `scp` line)

**Review by bertptrs — APPROVED** [review]

**Comment by Riscky** [comment]
@OpsBotPrime merge

**Comment by OpsBotPrime** [comment]
Pull request approved for merge by @Riscky, rebasing now.

**Comment by OpsBotPrime** [comment]
Rebased as e677f8c25966ba490214771921354afc209ea3a4, waiting for CI …

**Pull request merged**